### PR TITLE
Django upgrade

### PR DESCRIPTION
**Pull request by trowik**

### CODE REVIEW

File: app/maildelay/urls.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import include, url
+from django.urls import include, re_path
 
 
 urlpatterns = [
-    url(r'^', include('mails.urls')),
+    re_path(r'^', include('mails.urls')),
```
Comment (winged) [inline app/maildelay/urls.py]: why the change to `re_path`?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "2"
```
Comment (winged) [inline docker-compose.yml]: This is unrelated to Django :)

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Django==1.11.22 # pyup: >=1.11,<1.12
+Django==2.0.4
```
Comment (winged) [inline requirements.txt]: Why not 2.2? Do all the dependencies still work with this django version?

File: docker-compose.yml (hunk 2 above)
Comment (trowik) [reply docker-compose.yml]: I know, but I forgot to change it in the last PR and addes it here

File: requirements.txt (hunk 3 above)
Comment (trowik) [reply requirements.txt]: Will upgrade to 3.x anyways, so I just chose this version.
Still have to check the dependencies

File: app/maildelay/urls.py (hunk 1 above)
Comment (trowik) [reply app/maildelay/urls.py]: Found in [django documentation](https://docs.djangoproject.com/en/2.0/ref/urls/#url), that url is likely to be deprecated.

File: requirements.txt (hunk 3 above)
Comment (trowik) [reply requirements.txt]: Dependencies still work

File: app/maildelay/urls.py (hunk 1 above)
Comment (winged) [reply app/maildelay/urls.py]: Nice, TIL!
